app/workers/commit_monitor_handlers/commit_range/rubocop_checker.rb

```diff
@@ -39,7 +39,10 @@ def rubocop_comments
 
   def replace_rubocop_comments
     logger.info("Updating PR #{pr_number} with rubocop comment.")
-    GithubService.replace_comments(fq_repo_name, pr_number, rubocop_comments) do |old_comment|
+
+    status = results["files"].select { |f| f["offenses"].any? }.empty?
```

Comment:
Can you just do `status = results["files"].any? { |f| f["offenses"].any? }` ?

Reply:
It works exactly the opposite. I will have to negate it as
```ruby
status = !results["files"].any? { |f| f["offenses"].any? }
```
otherwise I will have to remake the rest of the code. Do you agree with the negation?

Reply:
if you're looking for the negation, then you can also do

```ruby
results["files"].none? { |f| f["offenses"].any? }
```